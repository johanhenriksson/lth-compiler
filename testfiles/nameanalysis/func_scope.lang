function a(int one) { 
    int one;
    int two;
    three = 5; // 2x error
}

function b() {
    int one; 
    one = two; // 2x error
}
