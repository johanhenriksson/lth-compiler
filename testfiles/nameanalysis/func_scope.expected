Error at line 4,5: symbol 'three' is not declared
Error at line 9,5: cannot assign 'one' of type int, got <unknown>
Error at line 9,11: symbol 'two' is not declared
